dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -34,6 +34,16 @@ export JQ_BINARY_DOWNLOAD_URL=${JQ_BINARY_DOWNLOAD_URL:-${DEFAULT_JQ_BINARY_DOWN
 DEFAULT_CHE_MULTIUSER="false"
 export CHE_MULTIUSER=${CHE_MULTIUSER:-${DEFAULT_CHE_MULTIUSER}}
 
+DEFAULT_CHE_DEPLOY=false
+export CHE_DEPLOY=${CHE_DEPLOY:-${DEFAULT_CHE_DEPLOY}}
+CHE_DEPLOYED=false
```

Comment:
what is the difference between
`CHE_DEPLOY` and `CHE_DEPLOYED`

Reply:
CHE_DEPLOYED is used so that if replace stacks is run before deploy che it will not run. Example  is when CLI arguments are `--replace-stacks --deploy-che`. However it could run twice with  `--deploy-che --replace-stacks` so I will correct this by adding a variable `STACKS_REPLACED` to ensure the replace stack is only ran once.

CHE_DEPLOY is used so that replace stacks will now if CLI argument `--deploy-che` is being used. This allows replace stack to be run without deploying.